go/vt/tabletserver/endtoend/compatibility_test.go

```diff
@@ -18,7 +18,7 @@ import (
 var point12 = "\x00\x00\x00\x00\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\x00@"
 
 func TestCharaterSet(t *testing.T) {
-	qr, err := framework.NewClient().Execute("select * from vitess_test where intval=1", nil)
+	qr, err := framework.NewClient().Execute("select * from vitess_test where intval=1 order by charval collate utf8_general_ci asc", nil)
```

Comment:
Let's revert this test change. I'm now convinced that `collate` doesn't affect character escaping. There's no need to change this.